.github/workflows/unit-tests.yml

```diff
@@ -12,11 +12,8 @@ jobs:
           # with default Python bridge
```

Comment:
Default is the C bridge in this branch, right?

Reply:
Right, I dropped the comment. There's just one bridge.